metrics/time/launch_times.sh

```diff
@@ -197,6 +230,125 @@ init () {
 	check_images "$IMAGE"
 }
 
+# Computes the average of the data
+calc_avg_array() {
+    result_arr=("$@")
+    avg=0
+
+    [ -z "$result_arr" ] && die "List of results was not passed to the calc_avg_array() function when trying to calculate the average result".
+
+    sum=$(IFS='+'; echo "scale=4; ${result_arr[*]}" | bc)
+    avg=$(echo "scale=$CALC_SCALE; $sum / $num_iters" | bc)
+    printf "%.0${CALC_SCALE}f" $avg
+}
+
+# Computes the stadard deviation of the data
+calc_sd_array() {
+    data=("$@")
+    sum_sqr_n=0
+    LSCALE=13
```

Comment:
@dborquez why we have this value?

Reply:
In the middle of calculating a statistic, we should avoid trimming or rounding numbers because the fractions are very small,
but once we have a result, we can limit the number of decimal digits.